include/taskolib/Context.h

```diff
@@ -40,15 +40,25 @@
 namespace task {
 
 /**
- * A VariableValue is a variant over several data types Lua can understand (long long, double, std::string, bool).
+ * Lua uses certain types to interface with C++.
+ *
+ * If these types are used the data comes from or shall go to the Lua engine.
+ */
+using LuaInteger = LUA_INTEGER;
+using LuaFloat = LUA_NUMBER;
+using LuaString = std::string;
+using LuaBool = bool;
```

Comment:
I guess this does not render nicely with Doxygen :grimacing:
Improvement ideas? (I'm a Doxygen dyslexic)